src/utils/export-module-completion.js

```diff
@@ -0,0 +1,78 @@
+const ES6_IMPORT_FROM_REGEX = /import\s+.*\s+from\s+['"](.*)['"]/;
+const CJS_IMPORT_FROM_REGEX = /require\(['"](.*)['"]\)/;
+
+const path = require('path');
+
+const escapeRegExp = require('lodash.escaperegexp');
+const { parseModule, parseFile } = require('esm-exports');
+
+const getRealExportPrefix = (prefix, line) => {
+  try {
+    const es6ExportPrefixRegex = new RegExp(`(?:import\\s+{|import)\\s+((?:.+?)*${escapeRegExp(prefix)})`);
+    const cjsExportPrefixRegex = new RegExp(`(?:var|const|let)\\s+\\{((?:.+?)*${escapeRegExp(prefix)})`, 'i');
+    const exportMatch = es6ExportPrefixRegex.exec(line) || cjsExportPrefixRegex.exec(line);
+    if (!exportMatch) {
+      return false;
+    }
+    const exportPrefix = exportMatch[1];
+    return exportPrefix.substring(exportPrefix.lastIndexOf(',') + 1).trim();
+  } catch (e) {
+    return false;
+  }
+}
+
+const getImportModule = (textRow) => {
+  try {
+    let importMatch = ES6_IMPORT_FROM_REGEX.exec(textRow) || CJS_IMPORT_FROM_REGEX.exec(textRow);
+    if (!importMatch) {
+      return false;
+    }
+    return importMatch[1];
+  } catch (e) {
+    return false;
+  }
+}
+
+const getExports = (activePanePath, prefix, importModule) => {
+  return importModule[0] !== '.' ?
+      lookupGlobal(importModule, activePanePath) :
+      lookupLocal(importModule, activePanePath);
+};
+
+const lookupGlobal = (importModule, activePanePath) => {
+  const projectPath = getProjectPath(activePanePath);
+  return parseModule(importModule, {dirname: projectPath})
+    .then(results => {
+      if (results.length > 0) {
+        return results.map(entry => entry.name);
+      }
+      return []; //TODO handle cjs global file
+    });
+};
+
+const getProjectPath = (activePanePath) => {
+  const [projectPath] = atom.project.relativizePath(activePanePath);
+  return projectPath;
+}
+
+const lookupLocal  = (importModule, activePanePath) => {
+  const filePath = activePanePath.substring(0, activePanePath.lastIndexOf('/'));
+  return parseFile(importModule, {dirname: filePath})
+    .then(results => results.length > 0 ?
+      results.map(entry => entry.name) :
+      lookupCommonJs(importModule, filePath))
+    .catch(() => {
+      return lookupCommonJs(importModule, filePath);
+    });
+};
+
+const lookupCommonJs = (importModule, projectPath) => {
+  const absoluteFile = path.resolve(projectPath, `${importModule}.js`);
+  return Object.keys(require(absoluteFile));
```

Comment:
I've come to a road-block.

Requiring a obsolute path in linux works like a charm.

But doesn't work on Windows. ARGH. Anyone knows why?